secure_time/secure_time_impl.c

```diff
@@ -402,11 +402,34 @@ int32_t secure_time_set_impl(uint64_t new_time)
     return SECURE_TIME_SUCCESS;
 }
 
+#define SECURE_TIME_MAX(A, B)   ((A) > (B) ? (A) : (B))
+
 uint64_t secure_time_get_impl(void)
 {
     uint64_t boot_time = secure_time_get_boot_time();
     uint64_t secs_since_boot = secure_time_get_seconds_since_boot();
-    return (boot_time > 0) ? (boot_time + secs_since_boot) : 0;
+
+    // If boot_time is valid (not 0), we can return boot_time + secs_since_boot as current time.
+    // Otherwise, the best estimation we have is the latest between the RTC and the stored time.
+    if(boot_time > 0) {
+        return boot_time + secs_since_boot;
+    }
```

Comment:
Note, run astyle please and send PR for the feature branch fixing the style for this secure-tiem (line 417 here would definitely get an error). Not certain why astyle did not fail in Travis, ignored?

Reply:
Or if you got time to fix it asap, we will wait with CI.

Reply:
ahh the missing space?

Reply:
Ill do it now

Reply:
Done

Reply:
Else should be on line 416. I just wanted to emphasize to be running astyle on features

Reply:
Can you refer us to the relevant astyle to use (for next time)?

Reply:
Use .astylerc file we have in the root and run `astyle --options=.astylerc` for code files